1-js/12-generators-iterators/1-generators/article.md

```diff
@@ -388,36 +390,36 @@ alert( generator.next(4).value ); // "3 * 3?"
 alert( generator.next(9).done ); // true
 ```
 
-The execution picture:
-
+Przebieg wykonania:
 ![](genYield2-2.png)
 
-1. The first `.next()` starts the execution... It reaches the first `yield`.
-2. The result is returned to the outer code.
-3. The second `.next(4)` passes `4` back to the generator as the result of the first `yield`, and resumes the execution.
-4. ...It reaches the second `yield`, that becomes the result of the generator call.
-5. The third `next(9)` passes `9` into the generator as the result of the second `yield` and resumes the execution that reaches the end of the function, so `done: true`.
+1. Pierwsze `.next()` rozpoczyna wykonanie... Dochodzi do pierwszego `yield`.
+2. Rezultat zostaje przekazany do kodu zewnętrznego.
+3. Drugi `.next(4)` przekazuje `4` z powrotem do generatora jako rezultat pierwszego `yield`, a następnie kontynuuje wykonanie.
+4. ...Wykonanie dochodzi do drugiego `yield`, ten staje sie rezultatem wywołania generatora.
+5. Trzeci `next(9)` przekazuje `9` do generatora, jako rezultat drugiego `yield`, następnie kontynuuje wykonywanie, które dochodzi do końca funkcji, a więc `done: true`.
 
-It's like a "ping-pong" game. Each `next(value)` (excluding the first one) passes a value into the generator, that becomes the result of the current `yield`, and then gets back the result of the next `yield`.
+To przypomina gre w "ping-pong". Każdy `next(value)` (za wyjątkiem pierwszego) przekazuje wartość do generatora, która staje sie rezultatem aktualnego `yield`, a następnie przekazuje rezultat następnego `yield`.
 
 ## generator.throw
 
-As we observed in the examples above, the outer code may pass a value into the generator, as the result of `yield`.
+Jak zauważyliśmy w przykładach wyżej, kod zewnętrzny może przekazać wartość do generatora, jako rezultat `yield`.
+As we observed in the examples above, the outer code may pass a value into the generator, as the result of yield.
```

Comment:
Pozostawiłeś angielską wersję w linii 407 "As we observed in the examples above, the outer code may pass a value into the generator, as the result of yield."